src/Mod/TechDraw/App/PreCompiled.h

```diff
@@ -54,14 +54,16 @@
 
 // Qt
 #include <QDomDocument>
-#include "QDomNodeModel.h"

```

Comment:
You can completely remove QDomNodeModel.h, QXmlQuery and QXmlResultItems. We do not use it any more, neither for Qt5 nor for Qt6.